### PR TITLE
[plt-6669] tests for Toolbar component

### PR DESCRIPTION
**Pull request by licarm**
## Description
tests for Toolbar component

## Where to Start
Toolbar.test.jsx

### CODE REVIEW

File: packages/riipen-ui/src/components/Toolbar.test.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Toolbar />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Toolbar.test.jsx:23]: Just a shout out to @licarm , Imo and you have been doing the snapshot test slightly differently from PR to PR. Sometimes we put this as part of the default props tests, and sometimes we make it a stand alone test like this.

We should be consistent. My 2 cents is that I like it as a stand alone test like this.

File: packages/riipen-ui/src/components/Toolbar.test.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  describe("children prop", () => {
+    it("renders given children inside the div element", () => {
+      const child = <h5>hello</h5>;
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Toolbar.test.jsx:27]: Is it me you're looking for?

**Review by jordanell — APPROVED** [review]
Great job!

File: packages/riipen-ui/src/components/Toolbar.test.jsx (hunk 1 above)
Comment (imogenem) [reply packages/riipen-ui/src/components/Toolbar.test.jsx:23]: We'll switch to a stand alone snapshot test!

**Pull request merged**